PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -109,6 +109,8 @@
         pfRelIso04_all = Var("(pfIsolationR04().sumChargedHadronPt + max(pfIsolationR04().sumNeutralHadronEt + pfIsolationR04().sumPhotonEt - pfIsolationR04().sumPUPt/2,0.0))/pt",float,doc="PF relative isolation dR=0.4, total (deltaBeta corrections)"),
         tightCharge = Var("?(muonBestTrack().ptError()/muonBestTrack().pt() < 0.2)?2:0",int,doc="Tight charge criterion using pterr/pt of muonBestTrack (0:fail, 2:pass)"),
         isPFcand = Var("isPFMuon",bool,doc="muon is PF candidate"),
+        isglobal = Var("isGlobalMuon",bool,doc="muon is global muon"),
+        istracker = Var("isTrackerMuon",bool,doc="muon is tracker muon"),
```

Comment:
please change the variable names to `isGlobal`, `isTracker` to comply better with cms namings

Reply:
Sure, I just changed these two variables names. Thank you.